src/app/wallets/get-on-chain-txs.ts

```diff
@@ -0,0 +1,24 @@
+import { BTC_NETWORK, ONCHAIN_MIN_CONFIRMATIONS, SECS_PER_10_MINS } from "@config"
```

Comment:
I've been thinking about our file structure. I think its a bit confusing that we have both these 'helper' files on the same level as the 'use-cases'. This file for instance should be considered 'module private' (only visible from `app/wallets`). Perhaps we should start adding a subfolder for these to highlight the difference in the folder structure.

I'm thinking of a subfolder called `private` or `internal` or something like that. I don't like `helpers` or `utils` so much because - even though somewhat accurate those are overloaded words that will probably not serve the correct purpose.

Reply:
Makes sense, can address in a follow-up PR

Reply:
Addressed: #1630